src/components/TooltipTrigger.js

```diff
@@ -108,10 +121,14 @@ class Tooltip extends Component {
     return createPortal(
       div({
         ref: this.element,
-        style: { ...styles.tooltip, transform: `translate(${finalPos.left}px, ${finalPos.top}px)` }
+        style: (type === 'light') ?
+          { ...styles.jobStatus, transform: `translate(${finalPos.left}px, ${finalPos.top}px)` } :
+          { ...styles.tooltip, transform: `translate(${finalPos.left}px, ${finalPos.top}px)` }
       }, [
         children,
-        svg({ viewBox: '0 0 2 1', style: { ...getNotchPosition(), ...styles.notch } }, [
+        svg({
+          viewBox: '0 0 2 1', style: (type === 'light') ? { ...getNotchPosition(), ...styles.noNotch } : { ...getNotchPosition(), ...styles.notch }
+        }, [
```

Comment:
Rather than making the notch invisible, it would be simpler to just not render it at all in the 'light' case.